snuba/migrations/operations.py

```diff
@@ -17,8 +18,21 @@ def __init__(self, storage_set: StorageSetKey):
     def storage_set(self) -> StorageSetKey:
         return self._storage_set
 
-    def execute(self, local: bool) -> None:
-        cluster = get_cluster(self._storage_set)
+    def execute(self, local: bool, partition_id: Optional[int]) -> None:
+        if (
+            partition_id is not None
+            and self._storage_set.value not in settings.PARTITIONED_STORAGE_SETS
```

Comment:
Are we going to have a `migrate_all` command that executes all migrations on all slices ? 
Normally the process for self hosted installs is to run bootstrap (or really any other command), that executes all the pending migrations. 

Since slices will be something people can set up, they should be taken into account automatically when moving to a new version.